src/harmonic.jl

```diff
@@ -0,0 +1,48 @@
+"""
+    harmonic(n)
+
+ Calculates harmonic numbers
+   e.g., see http://mathworld.wolfram.com/HarmonicNumber.html
+
+## Arguments
+* `n::Integer`: index of the Harmonic number to calculate
+
+## Examples
+```jldoctest
+julia> harmonic(2)
+1.5
+```
+"""
+function harmonic(n::Integer)
+    # γ = euler_mascheroni_const = 0.577215664901532860606512090082402431042 # http://oeis.org/A001620
+    if n <=10
+        # get exact values for small n
+        return sum( 1.0./(1:n) )
```

Comment:
Have fixed. Very interesting reading. Not sure I understand everything yet, but I get the main point. 

It sounds like there are new things happening for v0.6, which could be cool.